definitions/pipelines/rnaseq.cwl

```diff
@@ -155,3 +163,10 @@ steps:
             bam: index_bam/indexed_bam
         out:
             [metrics, chart]
+    cgpbigwig_bamcoverage:
```

Comment:
In some coverage tools (bedtools for example), you need to specify special parameters to handle split reads, so that a read split across two exons doesn't get counted through the entire intron. Is that the case with this tool?  (Is that what the `-a  --overlap   Use overlapping read check` param is for?)  Can you grab a small test case, and confirm whether this is handling such a case properly?

Reply:
Yes you are right, the -a  --overlap checks for those overlapping reads. Should I add that?

Reply:
For the RNA-Seq calculations, yes, please!